add log-anomaly section to Logs documentation (#13390)
DataDog/documentation: content/en/logs/explorer/insights.md

---
title: Watchdog Insights for Log Explorer
kind: documentation
description: 'Rely on Watchdog Insights to surface anomalous logs within your search context.'
further_reading:
    - link: 'logs/explorer/group'
      tag: 'Documentation'
      text: 'Group queried logs'
    - link: 'logs/explorer/visualize'
      tag: 'Documentation'
      text: 'Create visualizations from logs'
    - link: '/logs/explorer/export'
      tag: 'Documentation'
      text: 'Export Log Explorer views'
---

## Overview

> Datadog Logs Explorer offers Watchdog Insights to highlight signals that stand out within your search context based on the time frame and tags.

Reviewer: ```suggestion
Datadog Log Explorer offers Watchdog Insights to highlight signals that stand out within your search context based on the time frame and tags.
```

In the Log Explorer, Watchdog Insights surface logs based on patterns and specific tags that have been flagged by algorithms such as log anomaly detection and error outliers detection.

{{< img src="logs/explorer/insights/insights-for-log-explorer.png" alt="Watchdog Insights in Log Explorer" style="width:100%;" >}}

### Log Anomaly Detection

Ingested logs are analyzed at the intake level where Watchdog performs aggregations on detected patterns as well as `environment`, `service`,`source` and `status`tags.
These aggregated logs are scanned for anomalous behaviors, such as the following:
- An emergence of logs with a warning or error status.
* A sudden increase of logs with a warning or error status.

The logs surface as Insights in the Log Explorer, matching the search context and any restrictions applied to your role.

> {{< img src="logs/explorer/insights/log-anomalies-light.mp4" alt="Log Anomalies as Insights" video="true">}}

Reviewer: ```suggestion
{{< img src="logs/explorer/insights/log-anomalies-light.mp4" alt="A user scrolling through the details of a specific insight" video="true">}}
```

---

> title: Logs from Watchdog Insights

Reviewer: ```suggestion
title: Watchdog Insights for Log Explorer
```

Reviewer: To be consistent with "Watchdog for RUM" and "Watchdog for Logs".

kind: documentation

> description: 'Rely on Watchdog Insights to surface meaningful signals within your current seach context.'

Reviewer: ```suggestion
description: 'Rely on Watchdog Insights to surface anomalous logs within your search context.'
```

> aliases:

Reviewer: Remove aliases, don't need it.

    - /logs/explorer/insights
further_reading:
    - link: 'logs/explorer/group'
      tag: 'Documentation'
      text: 'Group queried logs'
    - link: 'logs/explorer/visualize'
      tag: 'Documentation'
      text: 'Create visualizations from logs'
    - link: '/logs/explorer/export'
      tag: 'Documentation'
      text: 'Export Log Explorer views'
---

## Overview

> Insights are a selection of meaningful signals that standout within your current search context (time and tags).

Reviewer: ```suggestion
Datadog Logs Explorer offers Watchdog Insights to highlight signals that stand out within your search context based on the time frame and tags.
```

> In the Log Explorer they allow our machine learning engine, Watchdog, to surface logs (caracterized by their patterns or specific tags) that have been flagged by our data science capabilities (like Log Anomaly Detection or Error Outliers Detection).

Reviewer: ```suggestion
In the Log Explorer, Watchdog Insights surface logs based on patterns and specific tags that have been flagged by algorithms such as log anomaly detection and error outliers detection.
```

> In the Log Explorer they allow our machine learning engine, Watchdog, to surface logs (caracterized by their patterns or specific tags) that have been flagged by our data science capabilities (like Log Anomaly Detection or Error Outliers Detection).

Reviewer 2: ```suggestion
Insights in the Log Explorer allow Datadog's machine learning engine, Watchdog, to surface logs characterized by their patterns or tags, and are flagged by Datadog's data science offerings, including Log Anomaly Detection and Error Outliers Detection.
```

> {{< img src="logs/explorer/insights/insights-carousel.png" alt="Watchdog Insights in Log Explorer" style="width:100%;" >}}

Reviewer: Should this be log-anomalies.png?

Author: I don't think so, this is about the general introduction of Insights in log explorer (via the carousel).
If we want further clarity, how about insights-for-log-explorer.png ?

### Log Anomaly Detection

> Ingested logs are analyzed at the intake level where Watchdog performs aggregations on detected patterns as well as `environment`, `service`,`source` and `status`tags.

Reviewer: ```suggestion
Ingested logs are analyzed at the intake level where Watchdog Insights perform aggregations on detected patterns as well as `environment`, `service`,`source` and `status` tags.
```

Author: Watchdog Insights is the vehicule for surfacing interesting things in context (Log Explorer, traces Explorer etc.).

Here it is about computations and it's the work of the Machine Learning engine, Watchdog itself.

To make this distinction clearer, we are working on reframing these so as to have: Watchdog (for the engine), Insights powered by Watchdog, Alerts powered by Watchdog etc.

> These aggregated logs are then scanned for anomalous behavour:

Reviewer: ```suggestion
These aggregated logs are scanned for anomalous behaviors, such as the following:
```

> * There is an emergence of logs in statuses warning, error or above.

Reviewer: ```suggestion
- An emergence of logs with a warning or error status.
```

> * The count of Logs in statuses warning, error or above is showing a sudden increase.

Reviewer: ```suggestion
* A sudden increase of logs with a warning or error status.
```

> Ingested logs are analyzed at the intake level where Watchdog performs aggregations on detected patterns as well as `environment`, `service`,`source` and `status`tags.
> These aggregated logs are then scanned for anomalous behavour:
> * There is an emergence of logs in statuses warning, error or above.
> * The count of Logs in statuses warning, error or above is showing a sudden increase.

Reviewer 2: ```suggestion
Ingested logs are analyzed at the intake level. Watchdog performs aggregations on detected patterns as well as `environment`, `service`,`source`, and `status` tags.

These aggregated logs are scanned for anomalous behavior:

* There is an emergence of logs in statuses warning, error, or above.
* The count of Logs in statuses warning, error, or above is showing a sudden increase.
```

> They are then surfaced as Insights in the Log Explorer when they match your current search context as well as any restrictions applied to your role.

Reviewer: What is the role?

Author: Everyone has access to these Insights but users are still subject to RBACs in place. Didn't want to use the term "RBAC", not being sure if this is something users are familiar with

> They are then surfaced as Insights in the Log Explorer when they match your current search context as well as any restrictions applied to your role.

Reviewer 2: ```suggestion
The logs surface as Insights in the Log Explorer, matching the search context and any restrictions applied to your role.
```

> {{< img src="logs/explorer/insights/log-anomalies-light.gif" alt="Log Anomalies as Insights" style="width:100%;" >}}

Reviewer 2: Please upload this file as an MP4: We do not accept GIFs as they are not accessible.

> Clikcing on specific insight, you get access to a full description of the detected anomaly as well as the list of patterns contributing to it.

Reviewer: ```suggestion
Click on a specific insight to see the full description of the detected anomaly as well as the list of patterns contributing to it.
```

> Clikcing on specific insight, you get access to a full description of the detected anomaly as well as the list of patterns contributing to it.

Reviewer 2: ```suggestion
By clicking on an insight, you can see a complete description of the detected anomaly as well as the list of patterns contributing to it.
```

### Error Outliers

> To begin searching for logs in Log Explorer, see the [Log Search Syntax documentation][1] and read the [time frame documentation][2] for more details on custom time frames.

Reviewer: ```suggestion
For more information about searching logs in the Log Explorer, see [Log Search Syntax][1] and [Custom Time Frames][2].
```
